src/kvm_ioctls.rs

```diff
@@ -1,3 +1,4 @@
+//! A safe wrapper around the kernel's KVM interface.
```

Comment:
This change does not seem to be related to the PR.

Reply:
Removed.